tbx/core/models.py

```diff
@@ -143,7 +146,27 @@ def partner_logos(self):
         return []
 
     content_panels = Page.content_panels + [
-        FieldPanel("introduction"),
+        MultiFieldPanel(
+            [
+                FieldPanel(
+                    "hero_heading",
+                    heading="Heading",
+                    help_text=mark_safe(
+                        "Use markdown to format the text. e.g. **bold** for <b>bold</b>."
+                    ),
+                ),
+                FieldPanel(
+                    "hero_introduction",
+                    heading="Introduction",
+                    help_text=mark_safe(
```

Comment:
What happens if we have text that is marked as bold and italic?

Reply:
Would it actually make sense to do some conversion of the markup with beautiful soup here to get some custom output like <span class="purple">?

Reply:
Do you mean something like: it'll be `<b>some</b> <i>content</i>` in the editor, but the output to the rendered page will be `<span class="pink">some</span> <span class="purple">content</span>`?

I expect it'd be a bit complicated but I suppose that's something we could do. I'll try to see if it's easy to do.

Reply:
It was just a thought to avoid the front-end styling being a bit confusing, but don't spend a long time on it if it is fiddly.

Reply:
@helenb I made some tweaks ([7aa8031](https://github.com/torchbox/torchbox.com/pull/313/commits/7aa80319b078ece35a89ca701e577604a6c612ad)) so that output would look like:

```html
<span class="text-coral">some</span> <span class="text-nebuline">content</span>
or
<span class="text-coral"><span class="text-nebuline">this for bold & italic links</span></span>
```

The span conversion only happens for links so that the editor could maybe still use bold & italics on non-link text.

Reply:
Thanks that sounds sensible. Does this now warrant a test?

Reply:
Added in [72bcae5](https://github.com/torchbox/torchbox.com/pull/313/commits/72bcae54519373ad5ad7fa7255a854e2984f355b) :)